websocket/client/client.go

```diff
@@ -17,21 +17,23 @@ import (
 
 // ArgsWebSocketClient holds the arguments needed for creating a client
 type ArgsWebSocketClient struct {
-	RetryDurationInSeconds int
-	WithAcknowledge        bool
-	BlockingAckOnError     bool
-	URL                    string
-	PayloadConverter       websocket.PayloadConverter
-	Log                    core.Logger
+	RetryDurationInSeconds     int
+	WithAcknowledge            bool
+	BlockingAckOnError         bool
+	BlockingSendIfNoConnection bool
```

Comment:
as discussed, maybe rename to `DropMessagesIfNoConnection`

Reply:
renamed